stories/__tests__/SQFormInclusionList.stories.test.tsx

```diff
@@ -140,4 +140,97 @@ describe('SQFormInclusionList Tests', () => {
       expect(submitButton).toBeEnabled();
     });
   });
+
+  it("It should display the error message 'friends field must have at least 5 items' when less than 5 items are selected in the fieldArray", async () => {
+    render(<WithValidation />);
+
+    const submitButton = screen.getByRole('button', {name: /submit/i});
+
+    await waitFor(() => {
+      const errorText = screen.getByText(
+        /friends field must have at least 5 items/i
+      );
+      expect(errorText).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(submitButton).toBeDisabled();
+    });
+
+    // get one of the unchecked checkbox elements
+    const jaipal = screen.getByLabelText('Jaipal');
+
+    //click checkbox to make validation pass by adding a 5th item to the array
+    userEvent.click(jaipal);
+
+    //check that our validation message is displaying 'All fields completed'
+    await waitFor(() => {
+      const validText = screen.getByText(/All fields completed/i);
+      expect(validText).toBeInTheDocument();
+    });
+
+    // check to ensure submit button is properly enabled
+    await waitFor(() => {
+      expect(submitButton).toBeEnabled();
+    });
+
+    // uncheck a checkbox to make the validation fail (less than 5 boxes checked)
+    const joe = screen.getByLabelText('Joe');
+    userEvent.click(joe);
+
+    await waitFor(() => {
+      const errorText = screen.getByText(
+        /friends field must have at least 5 items/i
+      );
+      expect(errorText).toBeInTheDocument();
+    });
+
+    const jane = screen.getByLabelText('Jane');
```

Comment:
On line 179 we uncheck a checkbox and then we expect the expected error message.
I think right here we can expect the Submit button to be disabled and end the test, right?

I'm not following what clicking Jane does for us.

Reply:
There is a specific situation in which the error would only happen if you unchecked two items below the validation limit and this was meant to test the user unchecking two checkboxes. I may be doing a poor job of explaining but I demo it in the loom for more context

Reply:
Yea I saw the loom but didn't connect the dots. I'm following what you're testing against now